ci/docker/centos7.dockerfile

```diff
@@ -34,13 +29,24 @@ RUN curl -L https://github.com/apache/arrow/archive/refs/tags/apache-arrow-9.0.0
     cmake3 ../arrow-apache-arrow-9.0.0/cpp \
         -DARROW_JEMALLOC=OFF \
         -DARROW_SIMD_LEVEL=NONE \
+        -DARROW_WITH_ZLIB=ON \
         -DCMAKE_INSTALL_PREFIX=../arrow && \
     cmake3 --build . && \
     make install
 
+RUN python3 -m venv /venv
+RUN source /venv/bin/activate && \
+    pip install --upgrade pip && \
+    pip install build Cython pytest pytest-cython numpy
+ENV NANOARROW_PYTHON_VENV "/venv"
+
+# Locale required for R CMD check
+RUN localedef -c -f UTF-8 -i en_US en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+
 # For R. Note that arrow is not installed (takes too long).
 RUN mkdir ~/.R && echo "MAKEFLAGS = -j$(nproc)" > ~/.R/Makevars
-RUN R -e 'install.packages(c("blob", "hms", "tibble", "rlang", "testthat", "tibble", "vctrs", "withr"), repos = "https://cloud.r-project.org")'
+RUN R -e 'install.packages(c("blob", "hms", "tibble", "rlang", "testthat", "tibble", "vctrs", "withr", "bit64"), repos = "https://cloud.r-project.org")'
 RUN rm -f ~/.R/Makevars
```

Comment:
This is part of every  docker file with the minor difference of installing arrow or not, should that not be moved into a script? + use ppm to install arrow for speed?